teos/src/config.rs

```diff
@@ -117,7 +118,7 @@ pub struct Opt {
 /// - Defaults
 /// - Configuration file
 /// - Command line options
-#[derive(Debug, Deserialize, Clone, PartialEq, Eq)]
+#[derive(Debug, Deserialize, Clone, PartialEq, Eq, Serialize)]
```

Comment:
nit: Let's move `Serialize` near to `Deserialize`.

Reply:
This hasn't been resolved either. Please do not resolve conservation for suggestions that have not been either replied to or addressed